attributes/default.rb

```diff
@@ -3,7 +3,19 @@
 # Attributes for InfluxDB
 
 # By default, always installs the latest by specifying nil
-default['influxdb']['version'] = nil
+default['influxdb']['version'] = '0.13.0'
+default['influxdb']['install_type'] = 'package'
+
+default['influxdb']['download_urls'] = {
+  'debian' => 'https://dl.influxdata.com/influxdb/releases',
+  'rhel' => 'https://dl.influxdata.com/influxdb/releases'
+}
+
+# platform_family keyed download sha256 checksums
+default['influxdb']['checksums']['0.10.3-1']['debian'] = '106e4884d8c654f1e43b803b21acc86e6494cafc50a46bcdb4330d0a51bd7aea'
```

Comment:
Can we make this happen?  I know you want to follow elasticsearch as an example, but I also believe in building in whats needed and getting rid of cruft that won't be used.